packages/@aws-cdk/aws-ec2/lib/machine-image.ts

```diff
@@ -479,6 +488,13 @@ export enum AmazonLinuxKernel {
    * Standard edition
    */
   KERNEL5_X = 'kernel-5.10',
+
+  /**
+   * Default kernel version
+   *
+   * Automatically changes with each major kernel version update
+   */
+  KERNEL_DEFAULT = 'kernel-default',
```

Comment:
I think one of the problems with this whole `AmazonLinux` feature is that things are different depending on which generation you are using. This enum applies to all generations, but does `kernel-default`?